R/helpers.R

```diff
@@ -57,3 +57,27 @@ pprob_to_matrix <- function(pp, task) {
   colnames(y) <- task$class_names
   y
 }
+
+convert_ratio = function(pv, target, ratio, n) {
```

Comment:
I'm struggling to see how this works, can you please add a one-line comment explaining what's happening and in one of the learners add a test to show it works. From what I can tell the following should fail (but doesn't):

```R
convert_ratio(list(mtry = 2, mtry.ratio = 0.2), "mtry", "mtry.ratio", 5)
```